test/README.md

```diff
@@ -101,19 +101,9 @@ Each file contains a sample manifest for an image configuration and some
 metadata describing the request that created the manifest and all the content
 (packages, containers, ostree commits).
 
-To (re)generate these test cases use the tool `cmd/gen-manifests`.
-
-```
-go run ./cmd/gen-manifests
-```
-will generate all manifests using the default options.
-
-Manifest generation can be restricted to only some distributions,
-architectures, or image types using command line flags.
-
-The command uses the configurations in
-`tools/test-case-generators/format-request-map.json` and repositories defined
-in `tools/test-case-generators/repos.json`.
+Those are legacy now and all manifest testing is happening in
+github.com/osbuild/images - there is a similar `test/data/manifest` directory
+in this repo.
```

Comment:
Please see #3967 and the corresponding COMPOSER-2127 and COMPOSER-2122. 

From what I am reading on the above 2 tickets + 1 PR there are still multiple image types which aren't covered by osbuild/images and it is unclear (as of now) when they will be fully covered. 

Based on this we've removed duplicate test scenarios and kept ones which were only covered in the osbuild-composer repository. Based on this and the fact that the format between the 2 repositories has changed I cannot approve this PR. 

Let's discuss it more before introducing any more changes.

Reply:
Thanks for this.  I admit, I lost track of how the manifests in osbuild-composer were being used.

Reply:
Thank you! Based on this I will close this PR for now until the tests are really fully transferred into a single repository.